src/v2/style-guide/index.md

```diff
@@ -305,6 +305,165 @@ De notre expérience, il est toujours mieux de _toujours_ ajouter une clé uniqu
 
 
 
+### Éviter `v-if` avec `v-for` <sup data-p="a">essentiel</sup>
+
+**N'utiliser jamais `v-if` sur le même élément que `v-for`.**
+
+Il y a deux utilisations courantes qui seraient tentantes :
+
+- Pour filtrer des éléments dans une liste (par ex. `v-for="user in users" v-if="user.isActive"`) Dans ce cas, remplacez `users` pas ure nouvelle propriété calculée qui retourne votre liste filtrée (par ex. `activeUsers`).
```

Comment:
Pour filtrer des éléments dans une liste (par ex. `v-for="user in users" v-if="user.isActive"`).

Le point final